src/2020-01-02-documenting-rcpp-packages.Rmd

```diff
@@ -0,0 +1,151 @@
+---
+title: "Documenting Rcpp functions and classes in R packages"
+author: "Artem Sokolov"
+license: GPL (>= 2)
+tags: documentation
+date: 2020-01-02
+summary: Demonstrates how to add Roxygen blocks in .cpp source
+---
+
+## Introduction
+
+[Roxygen2](https://cran.r-project.org/web/packages/roxygen2/index.html) is a convenient way to document functions in R packages. Based on the [Doxygen](https://en.wikipedia.org/wiki/Doxygen) model, it parses relevant information from the comments and generates the corresponding `man/*.Rd` files. The major strength of this model -- besides not having to write `*.Rd` files by hand -- is that documentation ends up living right next to the functionality it describes, thus enabling easy maintenance and synchronization between the two.
+
```

Comment:
I usually wrap at 80 or 100 columns, but mostly because that's what my editor does anyway...   Easy to fix.